torch_xla/__init__.py

```diff
@@ -10,6 +9,7 @@
 
 XRT_RUN_SERVER_PROCESS = 'torch_xla.core._xrt_run_server'
 XRT_SERVER_REGEX = '^python3 -m {} [0-9]+$'.format(XRT_RUN_SERVER_PROCESS)
+XRT_CONFIG_ENV_VARS = ['XRT_TPU_CONFIG', 'XRT_DEVICE_MAP', 'XRT_WORKERS']
```

Comment:
currently if user sets `GPU_NUM_DEVICES` it will default to xrt:gpu, user have to overwrite that with additional `PJRT_DEVICE=GPU`. I guess we intentionally don't change that behavior there?

Reply:
Right. I left GPU out intentionally because we don't have the same easy way of knowing that the user intended XRT, and there are still some feature gaps on PJRT GPU.